src/sieve.jsn

```diff
@@ -222,8 +211,8 @@
   },
   "instagram": {
     "link": "^(?:(?:i\\.)?instagr(?:\\.am|am\\.com))/(?:\\w+/)?(?:p/|tv/){1,2}([^/?#]+).*",
-    "url": "https://www.instagram.com/p/$1/?__a=1",
-    "res": ":\nvar i,c,u\n$._=JSON.parse($._)\n$=$._.graphql.shortcode_media\ni=$.edge_sidecar_to_children\ni=i&&i.edges\nc=$.edge_media_to_caption.edges,c=c.length&&c[0].node.text,f=$.owner.full_name\nc='@'+$.owner.username+(f?' ('+f+')':'') + ' | ' + new Date($.taken_at_timestamp*1e3).toLocaleString()+' '+(c?' | '+c:'')\nif(i&&i.length) $=i.map(function(el,i){return [el.node.video_url || el.node.display_url, !i&&c]})\nelse $=[$.video_url || $.display_url, c]\nreturn $"
+    "url": "https://www.instagram.com/graphql/query/?query_hash=d4e8ae69cb68f66329dcebe82fb69f6d&variables={\"shortcode\":\"$1\"}",
```

Comment:
`d4e8ae69cb68f66329dcebe82fb69f6d` looks like a magic number that might change, but I'll trust that it works.